docs/devsetup/java/maven.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: default
+title: Maven Setup for Java
+grand_parent: Setup for Contributors
+parent: Java Setup
+---
+
+<!--
+© 2016 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html
+-->
+
+# Maven Setup for Java
+{: .no_toc }
+
+## Contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+
+Maven is a pretty standard build tool in the Java ecosystem with a very well-defined
+preferred way to organize the directory structure and execute builds.
+Thus, many IDEs for Java will have some level of support for Maven.
+The command line invocation is still, of course, the standard of truth for the build.
+
+## Installation
+
+Install Maven from your OS package manager.
+The minimum version is 3.2.5, as configured in the `<ICU>/icu4j/pom.xml` root POM file's maven-enforcer-plugin.
+The Maven page lists [system requirements](https://maven.apache.org/download.cgi),
+which includes a minimum of JDK 8.
+
+## IDE Setup
+
+Users of IDEs should familiarize themselves with the information about how to use Maven at
+the command line,
+which also includes information about Maven builds.
+
+> :point_right: **Note**: In most IDEs, whenever the Maven configurations change (`pom.xml` files),
+you will need to refresh your IDE project using the IDE's Maven plugin/functionality.
```

Comment:
Nitpick: maybe add an extra sentence, something like "the exact way to do that is IDE specific, and will be documented in the section for each IDE" (?)

Reply:
Ack.